runtime/criusupport/criusupport.cpp

```diff
@@ -592,6 +622,13 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			/* throw the pending exception */
 			goto wakeJavaThreads;
 		}
+
+		if (vm->checkpointState.isJdwpEnabled) {
+			acquireSafeOrExcusiveVMAccess(currentThread, vmFuncs, safePoint);
+			toggleSuspendOnJavaThreads(currentThread, TRUE, TRUE);
+			releaseSafeOrExcusiveVMAccess(currentThread, vmFuncs, safePoint);
+		}
+
```

Comment:
youill need to add a vm release/acquire in `j9gc_prepare_for_checkpoint` as well

```
	releaseVMAccess(vmThread);
	extensions->configuration->adjustGCThreadCountForCheckpoint(env);
	acquireVMAccess(vmThread);
```